lib/config.py

```diff
@@ -192,6 +192,10 @@ def _setup_config_parser_options(self):
         self.parser.add_argument('--log-size',
                                  help='Size in bytes above which the log file '
                                  'should rotate', type=int, default=2<<20)
+        self.parser.add_argument('--work-dir', '-w',
+                                 help='Directory used to store all artifacts '
+                                 'created during the process.',
+                                 default='build')
```

Comment:
The default is already in `config.yaml`, there's no reason to have it here too. This will soon be fixed for other parameters too.